### ISSUE TITLE
unstyled elements https://ci.appveyor.com/billing

### ISSUE THREAD

**Issue by the-j0k3r** [xt0rted/AppVeyor-Dark#3]
Hey found some unstyled elements.

https://ci.appveyor.com/billing

![capture](https://user-images.githubusercontent.com/31389848/38777929-739e1574-40a8-11e8-8509-b10c7a8f3b48.PNG)

I have been playing with these selectors and effect.
```css
	.medium-9 .columns {
		filter: invert(100%) !important;
	}
```

but tbh I dont like the results and the text in these areas are unreadable also.
![capture](https://user-images.githubusercontent.com/31389848/38777889-bd0df6b2-40a7-11e8-9dd4-d33764fe6700.PNG)

I had a look around all the pages and this seems the only bits.

I dislike the default style hover effect on the buttons, it feels like they are having a fit, but thats an issue for AV team I guess.

Once you leave https://ci.appveyor.com and goto https://www.appveyor.com/ everything is obviously not included and unstyled, but I dont often go there.

**Comment by the-j0k3r** [comment]
the fix for the nearly invisible text which is in many many areas like https://ci.appveyor.com/project/someorganization/project/settings/deployment is

add `.ng-binding` to
```css
    .top-bar-section .dropdown li label, .ng-binding {
        color: #606060 !important;
    }
```

**Comment by xt0rted** [comment]
Can you update to 1.0.2 and see how this page looks now? As mentioned in the PR my view is different than this, but I did make some changed to the inputs so I'm curious to see if that helps at all.

**Comment by xt0rted** [comment]
Thanks to the site using angular I was able to toggle the various billing sections. The pricing table should be fixed in 1.0.3.

**Comment by the-j0k3r** [comment]
I tried to keep the middle box header color unchanged, clearly its their intention to make it stand out.

That said your solution works with that caveat mentioned,though Im  unsure why your selectors and mine are so different. Is there a reason why you done it like you did?
